.ddev/config.yaml

```diff
@@ -0,0 +1,262 @@
+name: usher
+type: php
+docroot: ""
+php_version: "8.2"
+webserver_type: nginx-fpm
```

Comment:
I went with Symfony for the lando config. Not sure if this matters at this stage of the game. But I can change the Lando one to be consistent with the ddev setup if you prefer.

Reply:
@apotek I went with `php` because [DDEV doesn't have a Symfony type](https://ddev.readthedocs.io/en/latest/users/configuration/config/#type). Not sure it matters.